### PR TITLE
[ASM] EXPANDR-7915 - update pack README

### PR DESCRIPTION
**Pull request by johnnywilkes**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
Update Pack README only for https://jira-dc.paloaltonetworks.com/browse/EXPANDR-7915 and other tickets

## Description
Update Pack README only for https://jira-dc.paloaltonetworks.com/browse/EXPANDR-7915 and other tickets

## Must have
- [ ] Tests
- [X] Documentation

### CODE REVIEW

**Comment by johnnywilkes** [comment]
@ShirleyDenkberg , can you look over, please?

**Comment by johnnywilkes** [comment]
@capanw / @BigEasyJ , please review

File: Packs/CortexAttackSurfaceManagement/README.md [hunk 1]
```diff
@@ -49,10 +50,13 @@ Automated remediation is only possible when the right conditions are met.  These
   - Azure Compute Instance
   - GCP Compute Engine (VM)
   - On-prem asset protected with a Palo Alto Networks Firewall
+  - An asset that is not one of these above, but is protected by Cortex Endpoint Security (XSIAM/XDR)
```
Comment (ShirleyDenkberg) [inline Packs/CortexAttackSurfaceManagement/README.md:53]:
```suggestion
  - An asset that is not one of the above, but is protected by Cortex Endpoint Security (XSIAM/XDR)
```

**Comment by ShirleyDenkberg** [comment]
@maimorag Doc review completed.

**Comment by johnnywilkes** [comment]
@maimorag , Any ideas why the Setup Environment build stage is failing? Seeing same failure on another PR: https://github.com/demisto/content/pull/34147

**Review by BigEasyJ — APPROVED** [review]

**Comment by johnnywilkes** [comment]
@maimorag / @yucohen , i have fixed the "Setup Environment" issue.  This is just an update to Pack README so I don't believe I need release notes.  Please advise when possible

**Comment by maimorag** [comment]
Hey @johnnywilkes,
Adding a release note is not necessary in this case.
Is the PR ready for merging?

**Comment by johnnywilkes** [comment]
> Hey @johnnywilkes, Adding a release note is not necessary in this case. Is the PR ready for merging?

Yes, please

**Review by maimorag — APPROVED** [review]

**Review by maimorag — APPROVED** [review]

**Comment by maimorag** [comment]
Hey @johnnywilkes please merge from master :)

**Comment by johnnywilkes** [comment]
> Hey @johnnywilkes please merge from master :)

done

**Pull request merged**